index.html

```diff
@@ -4,104 +4,173 @@
   <meta charset="utf-8"/>
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-  <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" rel="stylesheet">
-  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/js/all.min.js" rel="stylesheet">
-  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
   <link href="./styles.css" rel="stylesheet">
   <link rel="icon" href="/favicon.ico" />
   <title>Wazo WebRTC Demo</title>
 </head>
+
 <body>
-    <!--  Authentication -->
-    <div class="row" id="authentication">
-      <div class="col-xs-12 col-sm-12 col-md-3 one col">
-        <form class="center" id="login-form">
-          <img class="wazo-logo" src="img/wazo-logo.svg" />
-          <input type="text" id="username" class="form-control login input-lg" placeholder="Login" required autofocus>
-          <input type="password" id="password" class="form-control login input-lg" placeholder="Password" required>
-          <input type="text" id="server" class="form-control login input-lg" placeholder="Server" required>
-	        <button class="btn btn-lg btn-primary btn-block btn-flat login" id="submit-login" type="submit">Login</button>
+  <!--  Authentication -->
+  <div class="row" id="authentication">
+    <div class="col-xs-12 col-sm-12 col-md-3 one col">
+      <form id="login-form">
+        <img class="wazo-logo" src="img/wazo-logo.svg" />
+        <div class="form-group">
+          <label for="username">Username</label>
+          <input type="text" id="username" name="username" class="login" placeholder="votrenom@mail.com" required autofocus>
+          <label for="password">Password</label>
+          <input type="password" id="password" name="password" class="login" placeholder="********" required>
+          <label for="server">Server URI</label>
+          <input type="text" id="server" name="server" class="login" placeholder="votreserver.io" required>
+          <button class="btn btn-lg btn-primary btn-block btn-flat login" id="submit-login" type="submit">Login</button>
           <div id="auth-error" class="alert alert-danger" role="alert"></div>
-        </form>
-      </div>
-      <div class="hidden-xs hidden-sm col-md-9 two col">
-        <div class="col-md-12"></div>
+        </div>      
+      </form>
+    </div>
+    <div class="hidden-xs hidden-sm col-md-9 two col">
+      <div class="col-md-12 right-quote">
+        <p>A certain king had a beautiful garden, and in the garden stood a tree.</p>
```

Comment:
Is it a @bogue's weird sentence ?

Reply:
SORRY I just noticed I had comments :( Not I think it's Sylvain's or anyone who did the mock ups !